sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md

```diff
@@ -119,6 +134,99 @@ foreach (KnowledgeBaseAnswer answer in response.Value.Answers)
 }
 ```
 
+## Examples - QuestionAnsweringProjectsClient
+
+The following examples show common scenarios using the `QuestionAnsweringProjectsClient` instance [created in this section](#create-a-questionansweringprojectsclient).
+
+### Create a new project
+
+To create a new project, you must specify the project's name and a create a `RequestContent` instance with the parameters needed to set up the project.
+
+```C# Snippet:QuestionAnsweringProjectsClient_CreateProjectAsync
+// Set project name and request content parameters
+string newProjectName = "NewFAQ";
+RequestContent creationRequestContent = RequestContent.Create(
+    new
+    {
+        description = "This is the description for a test project",
+        language = "en",
+        multilingualResource = false,
+        settings = new
+        {
+            defaultAnswer = "No answer found for your question."
+        }
+    }
+    );
+
+Response creationResponse = await client.CreateProjectAsync(newProjectName, creationRequestContent);
+
+// Projects can be retrieved as follows
+AsyncPageable<BinaryData> projects = client.GetProjectsAsync();
```

Comment:
As much as most of us would like to use async methods by default in samples, we tend to use sync methods since not everyone is familiar with async (especially `await foreach`). It's okay to have some async samples, but if doing so should have good reason and have a sync analog.

Reply:
I've changed all the methods in the readme to be Sync. The rest of the samples have both sync and async versions.

However, for the migration guide I decided to keep our client's methods async for the following two reasons:
- The old cognitive service client only has Async methods (as far as I can tell), so it makes them to have them be the same type.
- The method calls are all one liners anyway. No unusual syntax like `await foreach`.

Does this seem okay with you?

Reply:
That's a great point and, yes, I agree it makes sense to use async in the migration guide. Thanks for considering that!